src/main/java/net/dries007/tfc/types/DefaultRecipes.java

```diff
@@ -275,6 +277,57 @@ public static void onRegisterLoomRecipeEvent(RegistryEvent.Register<LoomRecipe>
         );
     }
 
+    @SubscribeEvent
+    public static void onRegisterQuernRecipeEvent(RegistryEvent.Register<QuernRecipe> event)
+    {
+        IForgeRegistry<QuernRecipe> r = event.getRegistry();
+
+        r.registerAll(
+            //Flux
+            new QuernRecipe(new ResourceLocation(MOD_ID, "boarx"), IIngredient.of("gemBorax"), new ItemStack(ItemPowder.get(Powder.FLUX), 6)),
+            new QuernRecipe(new ResourceLocation(MOD_ID, "chalk"), IIngredient.of("rockChalk"), new ItemStack(ItemPowder.get(Powder.FLUX), 2)),
```

Comment:
Minor nitpick: these four recipes can be one, and use the existing ore dict name `rockFlux`